rpc/documentation/api.md

```diff
@@ -1476,3 +1478,43 @@ The user input can be either a hexadecimal string or a mnemonic word list.
 - `InvalidArgument`: The input is invalid.
 
 **Stability:** Unstable
+
+#### `StartTicketPurchase`
+
+The `StartTicketPurchase` method enables ticket purchases for stake mining in the wallet.
+
+**Request:** `StartTicketPurchaseRequest`
+- `passphrase`: The private passphrase to unlock the wallet.
+- `account_name`: The name of the account being queried.
```

Comment:
We also want to refer to accounts by their number not by the name.  Although the name is unique at any one time, it is not guaranteed to uniquely refer to the same account at different points of time (for that we use the account number, which is immutable).